### PR TITLE
[runtime] Get mono/tests running on bitcode/arm

### PR DESCRIPTION
**Pull request by alexanderkyte**
This fixes up some bitrot on fullaot mono/tests test execution and disables a test that crashes the runtime on ARM and doesn't seem to be expected to be working.

### CODE REVIEW

File: mono/tests/Makefile.am [hunk 1]
```diff
@@ -353,7 +353,6 @@ BASE_TEST_CS_SRC=		\
 	bug-340662_bug.cs	\
 	bug-325283.2.cs	\
 	thunks.cs \
-	winx64structs.cs \
 	nullable_boxing.2.cs	\
```
Comment (vargaz) [inline mono/tests/Makefile.am]: Why is this removed ?

File: mono/tests/Makefile.am [hunk 2]
```diff
@@ -556,7 +555,6 @@ TEST_IL_SRC=			\
 	generic-array-exc.2.il	\
 	generic-valuetype-newobj2.2.il	\
 	generic-valuetype-newobj.2.il	\
-	generic-constrained.2.il	\
 	generic-type-load-exception.2.il	\
```
Comment (vargaz) [inline mono/tests/Makefile.am]: Ditto.

File: mono/tests/Makefile.am [hunk 3]
```diff
@@ -1641,7 +1643,6 @@ GSHARED_TESTS_SRC = \
 	generic-getgenericarguments.2.cs	\
 	generic-synchronized.2.cs	\
 	generic-delegate-ctor.2.cs		\
-	generic-constrained.2.cs	\
 	bug-431413.2.cs			\
```
Comment (vargaz) [inline mono/tests/Makefile.am]: Ditto.

File: mono/tests/Makefile.am (hunk 1 above)
Comment (alexanderkyte) [inline mono/tests/Makefile.am]: It's moved to the profile_disabled tests because it causes a crash inside the LLVM verifier.

Comment (vargaz) [inline mono/tests/Makefile.am]: Isn't this the master list of tests ? They don't need to be moved anywhere.

Comment (alexanderkyte) [inline mono/tests/Makefile.am]: Hmm, maybe I disabled it in the wrong place. I'll fix this.

**Comment by vargaz** [comment]
This looks ok, but it doesn't disable any tests anymore from tests/.

**Comment by alexanderkyte** [comment]
It disables some in mono/tests/gc-descriptors/Makefile.am

**Comment by alexanderkyte** [comment]
How does this all look?
